### PR TITLE
Update release info

### PR DESCRIPTION
**Pull request by thekp**
Resolves n/a

**Overall change:**
_Update docs/Simorgh-Release-Info.md with most read page details_

**Code changes:**

- _Add release dates for most read page for all services, STY for Gahuza and Zhongwen and update Photogallery date_
- _Update `scripts/simorghPages.js` to include mostReadPage_
- _run `node scripts/simorghPages.js` to update Release-info doc_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: scripts/simorghLaunchDates.js [hunk 1]
```diff
@@ -320,32 +359,37 @@ const simorghLaunchDates = {
...
     photoGalleryPage: '12th March 2020',
     storyPage: '',
+    mostReadPage: '28th May 2020',
   },
   zhongwenSimp: {
     articles: 'Sept 2019',
     frontPage: '31st March 2020',
     photoGalleryPage: '24th March 2020',
-    storyPage: '',
+    storyPage: '28th May 2020',
```
Comment (karinathomasbbc) [inline scripts/simorghLaunchDates.js]: Should this be in the ukchinaSimp section?

File: scripts/simorghLaunchDates.js [hunk 2]
```diff
@@ -320,32 +359,37 @@ const simorghLaunchDates = {
...
+    mostReadPage: '28th May 2020',
   },
   zhongwenTrad: {
     articles: 'Sept 2019',
     frontPage: '31st March 2020',
     photoGalleryPage: '24th March 2020',
-    storyPage: '',
+    storyPage: '28th May 2020',
```
Comment (karinathomasbbc) [inline scripts/simorghLaunchDates.js]: Should this be in the ukchinaTrad section?

Comment (thekp) [reply scripts/simorghLaunchDates.js]: yep 🤦

File: scripts/simorghLaunchDates.js (hunk 1 above)
Comment (thekp) [reply scripts/simorghLaunchDates.js]: good spot, I've updated in the latest commit

File: docs/Simorgh-Release-Info.md [hunk 3]
```diff
@@ -3,52 +3,52 @@ This table can then be generated using the following command: `node scripts/simo
...
....test.bbc.com/gahuza)<br/>[articles](https://www.test.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.test.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.test.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.test.bbc.com/gahuza/23111981)<br/>[STY](https://www.test.bbc.com/gahuza/23204135) | [home](https://www.stage.bbc.com/gahuza)<br/>[articles](https://www.stage.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.stage.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.stage.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.stage.bbc.com/gahuza/23111981)<br/>[STY](https://www.stage.bbc.com/gahuza/23204135) | [home](https://www.bbc.com/gahuza)<br/>[articles](https://www.bbc.com/gahuza/articles/cryd02nzn81o)<br/>[liveRadio](https://www.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.bbc.com/gahuza/amakuru-51710168)<br/>[PGL](https://www.bbc.com/gahuza/amakuru-43894701)<br/>[STY](https://www.bbc.com/gahuza/amakuru-52821373) | __Home__: 19th Feb 2020<br/>__Articles__: Sept 2019<br/>__Live Radio__: 10th Jan 2020<br/>__MAP__: 27th Feb 2020<br/>__PGL__: 12th March 2020<br/>__STY__: 28th May 2020 |
```
Comment (FK78) [inline docs/Simorgh-Release-Info.md:14]: `[STY](https://www.bbc.com/gahuza/amakuru-52821373)`

I think you removed the LIVE link.

Comment (thekp) [reply docs/Simorgh-Release-Info.md:14]: oh yeah, let me check the cypress settings for STY 👁️ 👁️

File: docs/Simorgh-Release-Info.md [hunk 4]
```diff
@@ -3,52 +3,52 @@ This table can then be generated using the following command: `node scripts/simo
...
...liveradio)<br/>[MAP](https://www.test.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.test.bbc.com/gahuza/23111981)<br/>[STY](https://www.test.bbc.com/gahuza/23307435)<br/>[mostRead](https://www.test.bbc.com/gahuza/popular/read) | [home](https://www.stage.bbc.com/gahuza)<br/>[articles](https://www.stage.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.stage.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.stage.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.stage.bbc.com/gahuza/23111981)<br/>[STY](https://www.stage.bbc.com/gahuza/23307435)<br/>[mostRead](https://www.stage.bbc.com/gahuza/popular/read) | [home](https://www.bbc.com/gahuza)<br/>[articles](https://www.bbc.com/gahuza/articles/cryd02nzn81o)<br/>[liveRadio](https://www.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.bbc.com/gahuza/amakuru-51710168)<br/>[PGL](https://www.bbc.com/gahuza/amakuru-43894701)<br/>[mostRead](https://www.bbc.com/gahuza/popular/read) | __Home__: 19th Feb 2020<br/>__Articles__: Sept 2019<br/>__Live Radio__: 10th Jan 2020<br/>__MAP__: 27th Feb 2020<br/>__PGL__: 12th March 2020<br/>__STY__: 28th May 2020<br/>__Most Read__: 28th May 2020 |
```
Comment (FK78) [inline docs/Simorgh-Release-Info.md]:
```suggestion
| gahuza | [home](http://localhost:7080/gahuza)<br/>[articles](http://localhost:7080/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](http://localhost:7080/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](http://localhost:7080/gahuza/amakuru-23257470)<br/>[PGL](http://localhost:7080/gahuza/amakuru-43894701)<br/>[STY](http://localhost:7080/gahuza/23307435)<br/>[mostRead](http://localhost:7080/gahuza/popular/read) | [home](https://www.test.bbc.com/gahuza)<br/>[articles](https://www.test.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.test.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.test.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.test.bbc.com/gahuza/23111981)<br/>[STY](https://www.test.bbc.com/gahuza/23307435)<br/>[mostRead](https://www.test.bbc.com/gahuza/popular/read) | [home](https://www.stage.bbc.com/gahuza)<br/>[articles](https://www.stage.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.stage.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.stage.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.stage.bbc.com/gahuza/23111981)<br/>[STY](https://www.stage.bbc.com/gahuza/23307435)<br/>[mostRead](https://www.stage.bbc.com/gahuza/popular/read) | [home](https://www.bbc.com/gahuza)<br/>[articles](https://www.bbc.com/gahuza/articles/cryd02nzn81o)<br/>[liveRadio](https://www.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.bbc.com/gahuza/amakuru-51710168)<br/>[STY](https://www.bbc.com/gahuza/amakuru-52821373)<br/>[PGL](https://www.bbc.com/gahuza/amakuru-43894701)<br/>[mostRead](https://www.bbc.com/gahuza/popular/read) | __Home__: 19th Feb 2020<br/>__Articles__: Sept 2019<br/>__Live Radio__: 10th Jan 2020<br/>__MAP__: 27th Feb 2020<br/>__PGL__: 12th March 2020<br/>__STY__: 28th May 2020<br/>__Most Read__: 28th May 2020 |
```

Comment (FK78) [reply docs/Simorgh-Release-Info.md]: The LIVE link was removed.

Comment (thekp) [reply docs/Simorgh-Release-Info.md]: yep, I just checked the cypress settings and we are missing links for LIVE for gahuza STY and ukchina

Comment (thekp) [reply docs/Simorgh-Release-Info.md]: resolved in the latest commit

**Review by FK78 — APPROVED** [review]

**Review by karinathomasbbc — APPROVED** [review]

File: cypress/support/config/settings.js [hunk 5]
```diff
@@ -5831,7 +5831,7 @@ module.exports = () => ({
       storyPage: {
         environments: {
           live: {
-            paths: ['/ukchina/trad/vert-aut-38314451'],
+            paths: ['ukchina/trad/52836902'],
```
Comment (sareh) [inline cypress/support/config/settings.js]:
```suggestion
            paths: ['/ukchina/trad/52836902'],
```

File: cypress/support/config/settings.js [hunk 6]
```diff
@@ -1330,7 +1330,7 @@ module.exports = () => ({
       storyPage: {
         environments: {
           live: {
-            paths: ['/gahuza/51946134'],
+            paths: ['gahuza/amakuru-52821373'],
```
Comment (sareh) [inline cypress/support/config/settings.js]:
```suggestion
            paths: ['/gahuza/amakuru-52690732'],
```

File: cypress/support/config/settings.js (hunk 5 above)
Comment (thekp) [reply cypress/support/config/settings.js]: Thanks Sareh 👍

File: cypress/support/config/settings.js (hunk 6 above)
Comment (thekp) [reply cypress/support/config/settings.js]: Resolved now in my latest commit: https://github.com/bbc/simorgh/pull/6691/commits/a468048767b0993a0ddcf24af1bb04308a43c046 :)

Comment (thekp) [reply cypress/support/config/settings.js]: good spot 💯 

before the fix 😭 
image

**Pull request merged**